src/main/java/jarvis/Jarvis.java

```diff
@@ -0,0 +1,179 @@
+package jarvis;
+
+import features.Task;
+import features.Todo;
+import features.Deadline;
+import features.Event;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Jarvis {
+    public static final String JARVIS_LOGO = "       __  ___      ____ _    __  ____ _____\n"
+            + "      / / /   |    / __ \\ |  / / /  _// ___/\n"
+            + " __  / / / /| |   / /_/ / | / /  / /  \\__ \\ \n"
+            + "/ /_/ / / ___ |_ / _, _/| |/ / _/ /_ ___/ / \n"
+            + "\\____(_)_/  |_(_)_/ |_(_)___(_)___(_)____/  \n";
+    public static final String LINE_W_NL = "____________________________________________________________\n";
+    public static final String LINE = "____________________________________________________________";
+    public static final String COMMAND_BYE = "bye";
+    public static final String COMMAND_LIST = "list";
+    public static final String COMMAND_DONE = "done";
+    public static final String COMMAND_TODO = "todo";
+    public static final String COMMAND_DEADLINE = "deadline";
+    public static final String COMMAND_EVENT = "event";
+
+    public static void main(String[] args) {
+        greetMessage();
+        Scanner in = new Scanner(System.in);
+        String userLine;
+        ArrayList<Task> taskList = new ArrayList<>();
+        Boolean isOn = true;
+        while (isOn) {
+            userLine = in.nextLine().trim();
+            if (userLine.toLowerCase().contains(COMMAND_BYE)) {
+                isOn = false;
+            }
+            handleUserLine(userLine, taskList);
+        }
+    }
+
+    private static void handleUserLine(String userLine, ArrayList<Task> taskList) {
+        String taskKeyword = extractTask(userLine);
+        try {
+            switch (taskKeyword) {
+            case COMMAND_LIST:
+                listTasks(taskList);
+                break;
+            case COMMAND_DONE:
+                doneTask(userLine, taskList);
+                break;
+            case COMMAND_TODO:
+                todoTask(userLine, taskList);
+                break;
+            case COMMAND_DEADLINE:
+                deadlineTask(userLine, taskList);
+                break;
+            case COMMAND_EVENT:
+                eventTask(userLine, taskList);
+                break;
+            case COMMAND_BYE:
+                byeMessage();
+                break;
+            default:
+                throw new IndexOutOfBoundsException();
```

Comment:
I am not sure I understand the purpose of this exception throwing. Why when the command keyword of the userline is not recognizable, the software throws an IndexOutOfBoundsException?